src/Core/Models/UserState.cs

```diff
@@ -1,46 +1,65 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿//using System;
+//using System.Collections.Generic;
+//using System.Linq;
+//using System.Text;
+//using System.Threading.Tasks;
+//using mbsoft.BrewClub.Data;
 
-namespace mbsoft.BrewClub.Models
-{
-	[Serializable]
-	public class UserState
-	{
+//namespace mbsoft.BrewClub.Models
+//{
+//	[Serializable]
+//	public class UserState
+//	{
 
-		public string Username { get; private set; }
+//		public string Email { get; private set; }
+		
+//		public bool IsAdmin { get; private set; }
 
-		public int UserId { get; private set; }
+//		public bool IsModerator { get; private set; }
 
-		public bool IsAdmin { get; private set; }
+//		public bool IsMember { get; private set; }
 
-		public bool IsAnonymous { get; private set; }
+//		public bool IsUser { get; private set; }
 
-		public string DisplayName { get; private set; }
+//		public bool IsAnonymous { get; private set; }
 
+//		public string DisplayName { get; private set; }
 
-		private UserState()
-		{
-		}
 
-		public static UserState Create(string username, bool isAdmin, string displayName, int userId)
-		{
-			return new UserState
-			{
-				DisplayName = displayName,
-				IsAdmin = isAdmin,
-				Username = username,
-				IsAnonymous = false,
-				UserId = userId,
-			};
-		}
+//		private UserState()
+//		{
+//		}
```

Comment:
Removed this class via comments because I wasn't sure if I could do away with it completely.  Forgot it was still here, will remove in next commit.